tests/test_p2p_daemon_bindings.py

```diff
@@ -560,13 +560,19 @@ async def handle_another_proto(stream_info, reader, writer):
 
     writer.close()
 
-    # test case: registering twice can override the previous registration
+    # test case: registering twice can not override the previous registration without balanced flag
```

Comment:
```suggestion
    # test case: registering twice can't override the previous registration without balanced flag
```

nit: In English, it's either `cannot` or `can't` :)